Automattic/woocommerce-subscriptions-core
# Clean up and polishing of Subscriptions Notifications

Fixes https://github.com/woocommerce/woocommerce-subscriptions/issues/4737

## Description







This should be a change that doesn't change the functionality at all. 

The following has changed:
- ~Added admin message after notification wasn't sent---maybe the approach is too butcher-y; Is there a better way to do this?~
  - Added order note when skipping sending the notification via manual trigger
- Improved docs for several classes and methods
- Renamed one method
- Refactored one method with early exits to make it more readable
- Removed accidentally committed PHPCS helper file
- Improved messaging to the user when they execute an action
- Fixed translations of singular/plural strings

## How to test this PR





Mostly just check the unit tests and check the tasks mentioned in the original issue have been fixed.

## Product impact
